site/nci/spack-compilers.yaml

```diff
@@ -1,11 +1,11 @@
 compilers:
   - compiler:
-      spec: oneapi@2024.0.2
+      spec: oneapi@2024.2.1
       paths:
-        cc: /apps/intel-tools/intel-compiler-llvm/2024.0.2/bin/icx
-        cxx: /apps/intel-tools/intel-compiler-llvm/2024.0.2/bin/icpx
-        f77: /apps/intel-tools/intel-compiler-llvm/2024.0.2/bin/ifx
-        fc: /apps/intel-tools/intel-compiler-llvm/2024.0.2/bin/ifx
+        cc: /apps/intel-tools/intel-compiler-llvm/2024.2.1/bin/icx
```

Comment:
A first quick question: would it make sense to switch to 2025.0.4?

Reply:
done